programmers/난이도별/level01.문자열_내_p와_y의_개수/suhyeon.js

```diff
@@ -0,0 +1,12 @@
+function suhyeon(s){   
+    var pExp = /p/gi;
+    var yExp = /y/gi;
+    
+    var pResult = s.match(pExp);
+    var yResult = s.match(yExp);
+    
+    if(pResult.length == yResult.length){
+        return true;
+    }
```

Comment:
```suggestion
    if (pResult == null || yResult == null) return false;
    else if(pResult.length == yResult.length) return true;
```
오 null 예외 추가하니까 테스트 통과되었습니다! 감사합니다!